apps/site/lib/site_web/controllers/schedule/timetable_controller.ex

```diff
@@ -67,49 +70,62 @@ defmodule SiteWeb.ScheduleController.TimetableController do
     end
   end
 
-  @spec trip_messages(Routes.Route.t(), 0 | 1) :: %{{String.t(), String.t()} => String.t()}
-  defp trip_messages(%Routes.Route{id: "CR-Haverhill"}, 0) do
+  @spec trip_messages(Routes.Route.t(), 0 | 1, Date.t()) :: %{
+          {String.t(), String.t()} => String.t()
+        }
+  def trip_messages(%Routes.Route{id: "CR-Haverhill"}, 0, _) do
     %{
       {"221", "place-WR-0067"} => "Via",
       {"221", "place-WR-0075"} => "Lowell",
       {"221", "place-WR-0085"} => "Line"
     }
   end
 
-  defp trip_messages(%Routes.Route{id: "CR-Haverhill"}, 1) do
+  def trip_messages(%Routes.Route{id: "CR-Haverhill"}, 1, _) do
     %{
       {"208", "place-WR-0085"} => "Via",
       {"208", "place-WR-0075"} => "Lowell",
       {"208", "place-WR-0067"} => "Line"
     }
   end
 
-  defp trip_messages(%Routes.Route{id: "CR-Lowell"}, 0) do
+  def trip_messages(%Routes.Route{id: "CR-Lowell"}, 0, _) do
     %{
       {"221", "place-NHRML-0218"} => "Via",
       {"221", "place-NHRML-0254"} => "Haverhill"
     }
   end
 
-  defp trip_messages(%Routes.Route{id: "CR-Lowell"}, 1) do
+  def trip_messages(%Routes.Route{id: "CR-Lowell"}, 1, _) do
     %{
       {"208", "place-NHRML-0254"} => "Via",
       {"208", "place-NHRML-0218"} => "Haverhill",
       {"208", "place-NHRML-0152"} => "-"
     }
   end
 
-  defp trip_messages(%Routes.Route{id: "CR-Franklin"}, 1) do
-    %{
-      {"790", "place-rugg"} => "Via",
-      {"790", "place-bbsta"} => "Fairmount"
-    }
+  def trip_messages(%Routes.Route{id: "CR-Fairmount"}, 1, date) do
+    case is_atleast_oct_21_2019(date) do
+      true ->
+        %{{"726", "place-FS-0049"} => "FRANK"}
+
+      false ->
+        %{}
+    end
   end
 
-  defp trip_messages(_, _) do
+  def trip_messages(_, _, _) do
     %{}
   end
 
+  def is_atleast_oct_21_2019(date) do
+    case Date.compare(date, ~D[2019-10-21]) do
```

Comment:
Could be rewritten as `do: Date.compare(date, ~D[2019-10-21]) != :lt`

Reply:
True, I prefer the case syntax in elixir code for conditions instead of a !=

I could do the case with a default _ arg for eq/gt but decided not to.